examples/BUILD.gn

```diff
@@ -897,7 +897,7 @@ if (is_win || is_android) {
 
 if (is_android) {
   rtc_android_library("webrtc_unity_java") {
-    java_files = [ "unityplugin/java/src/org/webrtc/UnityUtility.java" ]
+    java_files = [ "remoting/java/src/org/webrtc/UnityUtility.java" ]
```

Comment:
We have no `remoting` folder in webrtc folder structure. Better keep the original one.

Reply:
It seems I missed that one, thanks for pointing it out!